nginx/files/security.conf

```diff
@@ -1,2 +1,5 @@
 # Security tweaks for nginx
 server_tokens off;
+
+# Cache SSL Sessions to improve performance and security
+ssl_session_cache shared:SSL:50m;
```

Comment:
Anders werkt ie niet :/ Gaat dan mis icm SNI. Zolang SSL niet aan staat zou dit sowieso geen probleem moeten zijn. Kan hem dan wel weer uit de server config zelf halen want dat is dan inderdaad dubbel.